demos/demo-minimal-js/package.json

```diff
@@ -4,14 +4,16 @@
   "description": "HubSpot calling extension sdk demo minimal js",
   "private": true,
   "scripts": {
-    "build": "node scripts/support_node_17+.js webpack",
+    "build": "webpack",
     "build:gh": "npm ci && cross-env NODE_ENV=production npm run build",
-    "serve": "node scripts/support_node_17+.js webpack serve",
+    "serve": "webpack serve",
```

Comment:
- What are the use cases for the `serve` and the `start` command?
- For the `start` command, do we always want to build before serving? Asking because I thought the serve command would build.

Reply:
The `serve` command follows the latest documentation from https://github.com/webpack/webpack-dev-server. The `serve` command doesn't generate files in the `dist` folder however so we need to build in the start command.

Reply:
The difference would be being able to preview the `dist` folder files when starting the app vs serving. I had specified `npm run serve` to serve the index html from the `dist` folder where the files are built but I think it makes more sense to reference the `public` folder.

If we don't need to minify the files when starting the app, then the only difference between `serve` and `start` would be the `--open` command.

Reply:
Sorry if this is all very confusing!! The reason why I needed to build the files was so that we could serve the `demo-minimal-js.html` or `demo-react-ts.html` file that is generated using HtmlWebpackPlugin.